packages/paste-website/src/components/shortcodes/component-header/index.tsx

```diff
@@ -59,6 +64,19 @@ interface ComponentHeaderProps {
       };
     }
   ];
+  packageStatus?: [
+    {
+      node: {
+        data: {
+          Documentation: boolean;
+          Figma: string;
+          Design_committee_review: string;
+          Engineer_committee_review: string;
+          Code: string;
```

Comment:
Is `Documentation` the only boolean because that's the only one that has 2 options (yes/no) rather than 3 (yes/no/na), because there will always be docs for a given thing?

Reply:
Yeah. Documentation comes from Airtable where its a checked/unchecked value, which equates to true/false.